client/src/components/ListGroup/ListGroup.scss

```diff
@@ -55,11 +56,16 @@
 
 .list-group-item__thumbnail {
   width: 64px;
-  height: 64px;
+  height: auto;
   display: block;
   background: $gray-lighter;
   float: left;
-  margin: -12px 12px 0 #{0 - $grid-gutter-width / 2};
+  margin: -12px 12px -12px #{0 - $grid-gutter-width / 2};
+
+  > img {
+    max-height: 65px;
+    height: 100%;
```

Comment:
The mocks do not cater for when an image is too width or too short, or perhaps it's cropped on the sides (the car picture)
I do not agree that the picture should be stretched if it wasn't tall enough, are we able to have it vertically-aligned middle instead?